app/src/main/java/com/example/cartoon/MainActivity.kt

```diff
@@ -77,7 +89,10 @@ class MainActivity : ComponentActivity() {
 
         // Check if no image is selected
         if (drawable == null) {
-            Toast.makeText(this, "No image selected", Toast.LENGTH_SHORT).show()
+            FancyToast.makeText(
+                this@MainActivity,
+                "No image selected",FancyToast.LENGTH_LONG,
```

Comment:
You can define all the messages which is used in this file under string.xml file and read it back  using context. This improves code quality and makes the next code review for message chages.